dotcom-rendering/src/components/LiveblogNotifications.importable.tsx

```diff
@@ -0,0 +1,110 @@
+import { css } from '@emotion/react';
```

Comment:
This file is lifted and shifted from [FollowWrapper](https://github.com/guardian/dotcom-rendering/blob/8ed2707172c1f92689bed6f4829f0b8b6791eadf/dotcom-rendering/src/components/FollowWrapper.importable.tsx). This could be refactored in a follow-up PR, once we know this functionality works.